Automattic/wp-calypso
# state: split `dataGetters` into separate files

This PR is a follow-up to https://github.com/Automattic/wp-calypso/pull/48779 which added ~3kb to the `/me/privacy` section chunk.

#### Changes proposed in this Pull Request

* Split functions from `state/data-getters` into separate file
* Directly import `requestDpa` from `/me/privacy`

#### Testing instructions

* Go to `/me/privacy` and request a DPA
* Make sure all functions which used to live in `data-getters/index.js` are re-exported
